samples/tutorial/Tutorial4.md

```diff
@@ -8,9 +8,6 @@ To follow this tutorial, launch Android Studio and open this folder (`samples/tu
 
 Start from the implementation of `tutorial-3-complete` if you're skipping ahead.
 
-## Adding new todo items
-
-A gap in the usability of the todo app is that it does not let the user create new todo items. We will add an "add" button on the right side of the navigation bar for this.
```

Comment:
We never actually added UI elements or wired up "new TODO" through the Screen / LayoutRunner.  Instead of adding those steps and further increasing this Tutorial I just removed them.